Makefile

```diff
@@ -87,14 +114,10 @@ lint-fix-md:
 	npx remark . .github -o
 
 lint-go:
-	goimports -d $(shell git ls-files "*.go")
+	@echo goimports -d '**/*.go'
+	@goimports -d $(shell git ls-files "*.go")
 	revive -formatter stylish -config revive.toml ./...
 
 lint-fix-go:
-	goimports -d -w $(shell git ls-files "*.go")
-
-lint-proto:
-	protolint lint api/wharfapi
-
-lint-fix-proto:
-	protolint lint -fix api/wharfapi
```

Comment:
Were there issues with this?
If not then maybe better to keep it in and add it to iver-wharf/wharf-cmd/pull/128 as well instead?

Reply:
Oops, I mistakenly removed it in my copy-pasting frenzy.

Adding protolint to wharf-cmd should be done via dedicated issue: https://github.com/iver-wharf/wharf-cmd/issues/16